### PR TITLE
Add `nullOrBoolean` assert

### PR DESCRIPTION
**Pull request by diogotorres97**

### CODE REVIEW

File: src/asserts/null-or-boolean-assert.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  this.validate = value => {
+    if (value !== null && typeof value !== 'boolean') {
+      throw new Violation(this, value, { value: 'must_be_null_or_a_boolean' });
+    }
+
+    if (value === null) {
+      return true;
+    }
+
+    return true;
```
Comment (Americas) [inline src/asserts/null-or-boolean-assert.js:33]: Hmmm :thinking:

**Review by Americas — CHANGES_REQUESTED** [review]
Sooner or later we should just create an assert `nullOr`.

File: src/asserts/null-or-boolean-assert.js (hunk 1 above)
Comment (diogotorres97) [reply src/asserts/null-or-boolean-assert.js]: copy paste 😄

**Comment by diogotorres97** [comment]
> Sooner or later we should just create an assert `nullOr`.

@Americas https://github.com/uphold/validator.js-asserts/pull/149 ?

**Comment by Americas** [comment]
> > Sooner or later we should just create an assert `nullOr`.
> 
> @Americas #149 ?

Damn that's an old PR, but even at that time things were not improved just because nobody had time to work on maintenance :laughing: 

But yeah, that, I would probably make it more composable by allowing an array of asserts.

**Review by Americas — APPROVED** [review]

**Pull request merged**